rpc/namenode.go

```diff
@@ -145,25 +145,32 @@ func (c *NamenodeConnection) resolveConnection() error {
 	}
 
 	var err error
+	stackErrors := fmt.Errorf("resolveConnection failed. ")
 
 	if c.host != nil {
 		err = c.host.lastError
 	}
 
-	for _, host := range c.hostList {
+	for i, host := range c.hostList {
 		if host.lastErrorAt.After(time.Now().Add(-backoffDuration)) {
+			stackErrors = fmt.Errorf("%sAttempt %v: still backing off host %v after error" +
```

Comment:
I don't think `Attempt i` is really accurate, it's `Namenode i`, I don't think it's that useful to keep the whole thing.

Reply:
Agreed, `Attempt` is not accurate. Altered to `NN %v` to keep the message shorter.